File: ABC069/A.scala
object Main{
  def main(args:Array[String]) = {
    val list = scala.io.StdIn.readLine.split(" ").map(_.toInt).toList
    println((list(0) + 1) * (list(1) + 1) - list(0) + 1 - list(1) - list(1) - list(0) - 1)
  }
}

File: ABC069/A.java
import java.io.*;
import java.util.*;

class A{
    public void run(){
        BufferedReder br = new BufferedReder(new InputStreamReder(System.in));
        String[] arg = br.readLine().split(" ");

    }
    public static void main(String... args){
        new A().run();
    }
}
